### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by fbelzunc**
@reviewbybees esp @jtnord @escoem

### CODE REVIEW

**Comment by jtnord** [comment]
whilst I'm ok with this - why not document the trouble shooting in the wiki and point this page to the wiki for more information.   The wiki is normally looked at by users, and the repo by developers - and this information is usable by users!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+active-directory-plugin
```
Comment (oleg-nenashev) [inline README.md]: NIT: could be replaced by a human-readable name

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+================
+
+[![Build Status](https://jenkins.ci.cloudbees.com/job/plugins/job/active-directory-plugin/badge/icon)](https://jenkins.ci.cloudbees.com/job/plugins/job/active-directory-plugin/)
```
Comment (oleg-nenashev) [inline README.md]: Such embedded status is useless till the PR builder runs within the same job. Better to remove till we have multibranch

**Comment by oleg-nenashev** [comment]
Weak :-1: , because it causes the documentation duplication between Wiki and GtHub. Two instances have different info inside. My recommendation is to have docs in a single place

**Comment by fbelzunc** [comment]
@reviewbybees done

**Pull request merged**